CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.18.0 Release
+- Release Date: April 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ - ``Bot`` tags were added for bot accounts feature in server v5.10 and mobile v1.18, meaning that mobile v1.17 and earlier don't support the tags.
```

Comment:
@amyblais why is this necessary to put under compatibility? This seems like a new feature that should just go under improvements?

Reply:
It's a release note that v1.17 and earlier don't support bot tags, so people might think it's a bug.